src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -189,6 +189,7 @@ export default {
 		},
 
 		participant() {
+			// TODO: use participant mixin ??
```

Comment:
not sure why the below check is not `!this.$store.getters.getToken()` ??

Reply:
Yeah, the thing is this.token is not initialized or injected into this component.
It's only used after creating a conversation and then it contains the new token, independent from which conversation you are currently in.

So by switching to the mixin you need to make this.token a computed instead of a data attribute and the computed returns `this.$store.getters.getToken()` and all existing references of `this.token` need to change to `this.newToken` or something like this.

Reply:
When reading the code I had the impression that the logic is only here to detect whether there's a call in progress already, as you can create new conversations while being in a call, and when we do so we don't want to switch to the new conversation but stay.

And for that check we need "isInCall" which in turn needs "participant". But the current implementation (before this PR) would say "participant disconnected" even if a token was set, so "isInCall" would likely be false. I can test this to confirm.

Reply:
Okay, maybe we are talking about different things. My question is about the inverted negation. It should be `if (!this.$store.getters.getToken())  instead of `if (this.$store.getters.getToken())`

Reply:
regarding the computed prop, I now made the two mixins use the getter from the store, which reduces dependencies and potential side effects: https://github.com/nextcloud/spreed/pull/4672/commits/a38299001b6f89561c7002fe429489ae8998f48c

Reply:
ah right, so yeah that looks like a bug, maybe from copy paste and then replacing `this.token`